### PR TITLE
Refactor model merge

### PR DESCRIPTION
**Pull request by mattrobertsky**
Please review.

### CODE REVIEW

File: app/models/package.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * limitations under the License.
+ */
+
+import play.api.libs.json._
+
+package object models {
+
+  type Litres = BigDecimal
+
+  implicit val litresJson = new Format[Option[Litres]]() {
```
Comment (hmrclt) [inline app/models/package.scala]: Do we need these anymore?

File: app/controllers/LiabilityToolController.scala [hunk 2]
```diff
@@ -18,393 +18,439 @@ package controllers
...
-          case _ => Redirect(routes.LiabilityToolController.displayCopackedProduced())
+  def submitBrandsProduced(): Action[AnyContent] =
+    ActionWithSessionId.async { implicit request =>
+      litresForm.bindFromRequest().fold (
+        formWithErrors => {
+          Future(BadRequest(pages.amount(formWithErrors,"brands",
+            LiabilityRoutes.displayWhoDoYouManufactureForPage(),
+            LiabilityRoutes.submitBrandsProduced())))
+        },
+        validFromData => {
```
Comment (nkaraolis) [inline app/controllers/LiabilityToolController.scala]: Change to valid form data please :)

File: app/controllers/LiabilityToolController.scala [hunk 3]
```diff
@@ -18,393 +18,439 @@ package controllers
...
 import scala.concurrent.Future
 
 @Singleton
-class LiabilityToolController @Inject()(val messagesApi: MessagesApi,
-                                        implicit val configuration: Configuration) extends Controller with I18nSupport {
+class LiabilityToolController @Inject()(
+  val messagesApi: MessagesApi,
+  val sessionCache: SessionCache = connectors.LTCSessionCache,
+  implicit val configuration: Configuration
+) extends Controller with I18nSupport {
```
Comment (nkaraolis) [inline app/controllers/LiabilityToolController.scala]: Change Controller to be FrontendController

File: app/controllers/LiabilityToolController.scala [hunk 4]
```diff
@@ -18,393 +18,439 @@ package controllers
 
 import javax.inject.{Inject, Singleton}
 
-import connectors.KeyStoreConnector
```
Comment (nkaraolis) [inline app/controllers/LiabilityToolController.scala]: Can we run optimise imports to remove unused imports

File: app/controllers/LiabilityToolController.scala [hunk 5]
```diff
@@ -18,393 +18,439 @@ package controllers
...
-          Redirect(routes.LiabilityToolController.displayCommissioned())
-        }
-      }
+  def answers_=(amt: Answers)(implicit hc: HeaderCarrier) =
+    sessionCache.cache(answersKey, amt)
+
+  def updateAnswers(f: Answers => Answers)(implicit hco: HeaderCarrier): Future[Unit] =
+    answers(hco).flatMap { s => answers = f(s) }.map { _ => () }
+
+  implicit def hc(implicit request: Request[_]): HeaderCarrier =
```
Comment (nkaraolis) [inline app/controllers/LiabilityToolController.scala]: Remove this once the controller is using frontend controller

File: app/connectors/KeyStoreConnector.scala [hunk 6]
```diff
@@ -23,34 +23,7 @@ import uk.gov.hmrc.play.config.{AppName, ServicesConfig}
 import uk.gov.hmrc.play.http.logging.MdcLoggingExecutionContext._
```
Comment (nkaraolis) [inline app/connectors/KeyStoreConnector.scala]: Can we optimise imports in this file please

File: app/models/Answers.scala [hunk 7]
```diff
@@ -20,14 +20,19 @@ package models
   * Created by matt on 11/10/17.
```
Comment (nkaraolis) [inline app/models/Answers.scala]: Can we remove this comment please

File: app/controllers/LiabilityToolController.scala (hunk 4 above)
Comment (mattrobertsky) [reply app/controllers/LiabilityToolController.scala]: Done.

File: app/connectors/KeyStoreConnector.scala (hunk 6 above)
Comment (mattrobertsky) [reply app/connectors/KeyStoreConnector.scala]: Done.

File: app/controllers/LiabilityToolController.scala (hunk 3 above)
Comment (mattrobertsky) [reply app/controllers/LiabilityToolController.scala]: Done.

File: app/controllers/LiabilityToolController.scala (hunk 5 above)
Comment (mattrobertsky) [reply app/controllers/LiabilityToolController.scala]: Done.

File: app/controllers/LiabilityToolController.scala (hunk 2 above)
Comment (mattrobertsky) [reply app/controllers/LiabilityToolController.scala]: Done.

File: app/models/Answers.scala (hunk 7 above)
Comment (mattrobertsky) [reply app/models/Answers.scala]: Done.

**Pull request merged**